backend/characterconnect/api/game_manager.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import annotations
+from ctypes.wintypes import CHAR
```

Comment:
The ctypes module is normally reserved for converting between types in the language C, and Python types.

The type hint you're looking for is "str", even though it's a single letter it should still be hinted with a str, if you really wanted you could use Literal["a" | "b"] etc. to force it to be a single letter, but I think that's a bit overkill.